src/sessionTypes/machineAuth.spec.js

```diff
@@ -348,8 +348,8 @@ describe('sessionTypes/MachineAuth', function() {
         });
 
         it('should return two different promises', function() {
-          expect(promiseOne).to.equal(expectedPromiseOne);
-          expect(promiseTwo).to.equal(expectedPromiseTwo);
+          expect(promiseOne).to.deep.equal(expectedPromiseOne);
```

Comment:
Yes, this makes the test pass, but I don't see this as an actual fix. It goes against what that test case is supposed to check (i.e. how can it check that it is returning two different promises if it's a deep equality now?)

That said, thanks for looking at this, because it made me realize what the bug is:
```
          const audienceNameOne = faker.hacker.adjective();
          const audienceNameTwo = faker.hacker.adjective();
```

It's failing when the two audience names happen to be the same name@!! DUH. One can just be changed to like `faker.lorem.word();`

Reply:
ooo ok, good. I will revert this :)